.github/workflows/release.yml

```diff
@@ -5,9 +5,25 @@ on:
     tags: ["v*"]
 
 jobs:
+  npm_publish:
+    name: Publish to npm
+    runs-on: ubuntu-latest
+
+    if: github.actor == "nicolo-ribaudo"
+
+    steps:
+      - name: Checkout the new tag
+        uses: actions/checkout@v1.0.0
+      
+      - name: Publish to npm
+        run: make publish-ci
+        env:
+          NPM_TOKEN: ${{ secrets.NICOLO_NPM_TOKEN }}
```

Comment:
If this ends up working, we can use @babel-bot's token and add other the people who currently have publish access